week-3/B-array-some/exercise.js

```diff
@@ -12,6 +12,11 @@ var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
 
+var containNull = pairsByIndex.some(function(include) {
+  include === null;
+  console.log(" null value is found in this array");
+  return process.exit(1);
```

Comment:
I don't think this is quite right. We aren't checking if the value is actually null - we need to use an `if` statement for that. How could we fix that?